services/provider/server/server.go

```diff
@@ -1382,61 +1382,73 @@ func (s *OCSProviderServer) appendClientProfileKubeResources(
 	if rbdClientProfileName != "" {
 		rbdClientProfile := profileMap[rbdClientProfileName]
 		if rbdClientProfile == nil {
-			rbdClientProfile = &csiopv1a1.ClientProfile{
+			rbdClientProfile = &csiopv1.ClientProfile{
 				ObjectMeta: metav1.ObjectMeta{
 					Name:      rbdClientProfileName,
 					Namespace: consumer.Status.Client.OperatorNamespace,
 				},
-				Spec: csiopv1a1.ClientProfileSpec{
+				Spec: csiopv1.ClientProfileSpec{
 					CephConnectionRef: corev1.LocalObjectReference{Name: consumer.Status.Client.Name},
 				},
 			}
 			profileMap[rbdClientProfileName] = rbdClientProfile
 
 		}
-		rbdClientProfile.Spec.Rbd = &csiopv1a1.RbdConfigSpec{
+		rbdClientProfile.Spec.Rbd = &csiopv1.RbdConfigSpec{
 			RadosNamespace: rnsName,
+			CephCsiSecrets: &csiopv1.CephCsiSecretsSpec{
+				ControllerPublishSecret: corev1.SecretReference{
```

Comment:
@iPraveenParihar Is adding the provisioner secret part of the v1alpha1 to v1 migration or is it a separate change?